src/core-components/Slider/Slider.tsx

```diff
@@ -0,0 +1,18 @@
+import './Slider.less';
+
+import { Slider as AntSlider, SliderSingleProps } from 'antd';
+import React from 'react';
+
+const marks = {
+  0: '0%',
+  25: '25%',
+  50: '50%',
+  75: '75%',
+  100: '100%',
+};
+
```

Comment:
I think it' good to set the marks as props in Slider component in order to customize the marks whatever they want.